bitnami/odoo/templates/deployment.yaml

```
@@ -248,7 +249,7 @@ spec:
         - name: odoo-data
           {{- if .Values.persistence.enabled }}
           persistentVolumeClaim:
-            claimName: {{ .Values.persistence.existingClaim | default (include "odoo.fullname" .) }}
+            claimName: {{ (tpl .Values.persistence.existingClaim $) | default (include "common.names.fullname" .) }}
```

Does it make sense to use `|` ?
```suggestion
            claimName: {{ default (include "common.names.fullname" .) (tpl .Values.persistence.existingClaim $) }}
```

---

It's exactly the same thing

---

I know, not a question about functionality but about style. I don't have a strong opinion about any of them as far as they work.